index.html

```diff
@@ -52,6 +52,146 @@ <h2>I'm a Software Developer</h2>
                 </ul>
             </div>
         </div>
+        <div class="works-section">
+            <div id="works-section-title">
+                <h1>My Recent Works</h1>
+            </div>
+            <div id="separator"></div>
+            <div id="featured-project">
+                <img id="featured-project-image" alt="Featured project image" src="src/featured-project-image.png">
+                <h2 id="featured-project-title">Multi-Post Stories</h2>
+                <p id="featured-project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="featured-project-languages">
+                    <li id="lang1">css</li>
+                    <li id="lang2">html</li>
+                    <li id="lang3">bootstrap</li>
+                    <li id="lang4">Ruby</li>
+                </ul>
+                <button class="featured-project-button" type="button">See Project</button>
```

Comment:
- [ ] Nice job for implementing this button section to march the design. You can improve the user experience by adding more button styling: _Active,_ _Hover_ and _Pressed_